pages/index.tsx

```diff
@@ -1,19 +1,36 @@
 import { Box, Button, Container, Typography } from '@mui/material';
 import jwt from 'jsonwebtoken';
 import { NextPage } from 'next';
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import Cookies from 'universal-cookie';
 
 import createAxiosInstance from 'client/lib/axios';
 import { getDiscordOauthUrl } from 'client/lib/oauth';
 import { AUTH_COOKIE_NAME } from 'shared/constants';
 
+interface User {
+  id: number;
+  discord_name: string;
+}
+
 interface Props {
   isAuthed: boolean;
 }
 
 const Index: NextPage<Props> = props => {
   const [isAuthed, setIsAuthed] = React.useState(props.isAuthed);
+  const [users, setUsers] = useState<User[]>([]);
+
+  useEffect(() => {
+    const fetchUsers = async () => {
+      const axios = createAxiosInstance();
+      const response = await axios.get('/api/users');
+      const users = response.data.users;
+      setUsers(users);
+      console.log(users);
+    };
+    fetchUsers();
+  }, []);
```

Comment:
Making the call in `useEffect` will ensure that the API request only happens client-side, meaning we aren't taking advantage of Next's SSR capabilities. This should probably be moved to `getInitialProps`